index.js

```diff
@@ -64,12 +64,15 @@ function get(url, timeout, port, protocol, detailed) {
   protocol = protocol || 'https:';
 
   var options = getOptions(url, port, protocol);
+  
+  if (timeout)
+    options['timeout'] = timeout;
 
   return new Promise(function(resolve, reject) {
     var req = handleRequest(options, detailed, resolve, reject);
 
     if (timeout) {
-      req.setTimeout(timeout, function() {
+      req.on('timeout', ()=> {
```

Comment:
Minor style issue: space between the arrow and the parens.

Reply:
Fixed [here](https://github.com/johncrisostomo/get-ssl-certificate/pull/27/commits/54161b26ffbc9062e1f1db43024ab49f554f473f)